config/default.yaml

```diff
@@ -38,10 +38,17 @@ scope:
         - "Serbia"
         - "Switzerland"
     bounds:
-        x_min: -15.8 # in degrees east
+        x_min: -30 # in degrees east
```

Comment:
If you change the `default.yaml`, I think you should test whether the entire workflow still works. Alternatively, create another config file. Even then, it would be good to test the workflow.

The package conflicts should be fixed at the moment.

Reply:
sure thing. I haven't checked the functioning of the workflow in this repo on any of these PRs, I have to admit...